### PR TITLE
ci: update workflow

### PR DESCRIPTION
**Pull request by wep21**
## PR Type

- Improvement

## Pre-Review Checklist for the PR Author

**PR Author should check the checkboxes below when creating the PR.**

- [x] Assign PR to reviewer

## Checklist for the PR Reviewer

**Reviewers should check the checkboxes below before approval.**

- [x] Commits are properly organized and messages are according to the guideline
- [ ] (Optional) Unit tests have been written for new behavior
- [x] PR title describes the changes

## Post-Review Checklist for the PR Author

**PR Author should check the checkboxes below before merging.**

- [ ] All open points are addressed and tracked via issues or tickets

## CI Checks

- **Build and test for PR**: Required to pass before the merge.

### CODE REVIEW

File: .clang-format [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# Modified from https://github.com/ament/ament_lint/blob/master/ament_clang_format/ament_clang_format/configuration/.clang-format
```
Comment (kenji-miyake) [inline .clang-format]: You can add these config files by `sync-files` workflow.
But there is no problem if you add this manually this time.

Comment (kenji-miyake) [reply .clang-format]: To make the review easier, it might be a good idea to run `sync-files` once and merge that first, then rebase this PR.

Comment (wep21) [reply .clang-format]: Ah, I understand. Thank you for your instruction🙏 I will try it next time.

Comment (kenji-miyake) [reply .clang-format]: How about merging `sync-files` first like this?
https://github.com/autowarefoundation/autoware.universe/pull/311

Comment (wep21) [reply .clang-format]: I created https://github.com/tier4/ros2-project-template/pull/2. 🙏

File: .github/workflows/build-and-test-arm.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
+name: build-and-test-arm
+
+on:
+  schedule:
+    - cron: 0 19 * * * # run at 4 AM JST
+  workflow_dispatch:
+
+jobs:
+  build-and-test-arm:
+    runs-on: ARM64
```
Comment (KeisukeShima) [inline .github/workflows/build-and-test-arm.yaml]: Do we need an ARM64 job?
This repository is intended for use by Tier 4 projects, and we do not have a self-hosted runner for ARM64.

**Review by KeisukeShima — COMMENTED** [review]
Thanks for creating the PR!

File: .github/workflows/build-and-test-arm.yaml (hunk 2 above)
Comment (kenji-miyake) [reply .github/workflows/build-and-test-arm.yaml]: I've pointed it out in #2 and it was removed.

Comment (KeisukeShima) [reply .github/workflows/build-and-test-arm.yaml]: I see. Thank you!

**Comment by kenji-miyake** [comment]
@Mergifyio rebase
-> I haven't installed it in this repository. :cry:

**Comment by kenji-miyake** [comment]
Instead, 
@wep21 rebase

File: .pre-commit-config.yaml [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+      - id: clang-format
+
+  - repo: https://github.com/cpplint/cpplint
+    rev: 1.5.5
+    hooks:
+      - id: cpplint
+        args: [--quiet]
+        exclude: .cu
+
+exclude: .svg|control/trajectory_follower|control/trajectory_follower_nodes|common/autoware_auto_cmake|common/autoware_auto_common|common/autoware_auto_geometry|common/autoware_auto_tf2|common/autoware_testing|common/fake_test_node|common/had_map_utils|common/motion_common|common/motion_testing|common/time_utils|common/vehicle_constants_manager|common/autoware_auto_perception_rviz_plugin|common/osqp_interface|simulator/simple_planning_simulator|planning/freespace_planner|planning/astar_search|planning/costmap_generator
```
Comment (kenji-miyake) [inline .pre-commit-config.yaml]:
```suggestion
exclude: .svg
```

**Review by kenji-miyake — APPROVED** [review]

**Comment by kenji-miyake** [comment]
Thank you!

**Pull request merged**